plugin/backend_linux.go

```diff
@@ -294,12 +294,18 @@ func (pm *Manager) Remove(name string, config *types.PluginRmConfig) error {
 	}
 
 	id := p.GetID()
-	pm.pluginStore.Remove(p)
+
+	defer func() {
+		if err == nil || config.ForceRemove {
```

Comment:
you'll need to name the error return for this to work, or change `if err :=` to `if err = `. and define error above.